### PR TITLE
clarified maintenance mode info display

### PR DESCRIPTION
**Pull request by ftrotter**
The current verbiage makes it seem that the task will ONLY run in maintenance mode, which is not correct. 
This makes it perfectly clear what the task will do in maintenance mode

### CODE REVIEW

**Comment by qschmick** [comment]
This is a standard feature from Laravel (https://laravel.com/docs/5.6/scheduling#maintenance-mode). If we are going to update the text it should fall in line with the documentation. By default the jobs will not run in maintenance mode so having the else output seems redundant to include. Additionally, I would say we should have the text say something along the lines of "This job will still run during maintenance mode."

**Comment by ftrotter** [comment]
I am not proposing that clarity is needed on how Laravel works by default. I understand how maintenance mode works. However, you expose this as an option on your GUI. Your settings control this as a toggle, which means that when you view a given task you should be able to know what status it is in. 

Your current view of a given task makes the information about a decision made about the task using your interface (i.e. that the checkbox was not chosen) something that is hidden from the user. In our particular use case, all of our tasks should always run in maintenance mode. For us, not having something with that toggle on would be a dangerous error. But with your current display, that error would be an easy one to make. Because rather than saying something explicit about the state of the toggle, in one "toggle state" your current interface says nothing at all. 

To put it another way. If it is useful to tell a user that a given task will run in maintenance mode. Then it is hard to argue that it is not useful to tell a user that a given task will not run in maintenance mode. 

Having said that, if you believe that this feature would be better implemented using exactly the same phrasing as the documentation that you link to makes sense and I would be happy to make that change and re-submit.

**Review by qschmick — APPROVED** [review]
